src/main/java/gregtech/common/GT_Worldgenerator.java

```diff
@@ -507,6 +509,14 @@ public void run() {
             if (tChunk != null) {
                 tChunk.isModified = true;
 
+                if (debugWorldGen && usedOreMix == null) {
+                    for (val player : this.mWorld.playerEntities) {
+                        val message = String.format("No orevein selected for chunk: %s", tChunk.getChunkCoordIntPair());
```

Comment:
Ensure this does not spam log in release!

Reply:
I FIXED THIS
AND I CONFIRMED IT WORK
[divine code]